e2e/benchmarks/services_test.go

```diff
@@ -0,0 +1,276 @@
+		if instances {
+			name, id = fmt.Sprintf("svc_%d", i), "instance"
+		}
+		services[i] = testutil.TestService{
+			Name:    name,
+			ID:      id,
+			Address: "127.0.0.2",
+			Port:    int(baseport + i),
+			Tags:    []string{},
+		}
+	}
+	return services
+}
+
+// writes the config files out to disk, returns path to them and cleanup func
+func makeConfig(t testing.TB, addr string, tls bool) (string, func() error) {
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "stress")
+	cleanup := testutils.MakeTempDir(t, tempDir)
+	modDir := filepath.Join(tempDir, "module")
+	testutils.MakeTempDir(t, modDir)
+
+	configPath := filepath.Join(tempDir, configFile)
+	err := writeConfig(configPath, configContents(addr, tls))
+	require.NoError(t, err)
+	modulePath := filepath.Join(modDir, "main.tf")
+	err = writeConfig(modulePath, nullModuleContents())
+	require.NoError(t, err)
+
+	return configPath, cleanup
+}
+
+// write the (config) contents to the path
+func writeConfig(configPath, contents string) error {
+	f, err := os.Create(configPath)
+	if err != nil {
+		return nil
+	}
+	defer f.Close()
+	config := []byte(contents)
+	_, err = f.Write(config)
+	return err
+}
+
+// not used but handy to keep around for when you need to check the added
+// services or instances in consul
+func showMeServices(t *testing.T, srv *testutil.TestServer) {
+	u := fmt.Sprintf("http://%s/v1/agent/services", srv.HTTPAddr)
+	resp := testutils.RequestHTTP(t, http.MethodGet, u, "")
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println(string(b))
+	defer resp.Body.Close()
+}
+
+// returns templated contents of config file
+func configContents(address string, tls bool) string {
+	pwd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	return fmt.Sprintf(`
+log_level = "WARN"
+
+consul {
+  address = "%s"
+  tls {
+	  enabled = %t
+	  verify = false
+  }
+}
+
+driver "terraform" {
+	path = "%s"
+	backend "local" {}
+}
+
+terraform_provider "local" {}
+
+task {
+  name = "stress-test"
+  source = "../../tmp_stress/module"
+  providers = ["local"]
+  services = ["stress"]
+}
+
+`, address, tls, pwd)
+}
+
+// null resource, services module
+func nullModuleContents() string {
+	return `
+resource "null_resource" "address" {}
+
+variable "services" {
+  description = "Consul services monitored by Consul Terraform Sync"
+  type = map(
+    object({
+      id        = string
+      name      = string
+      kind      = string
+      address   = string
+      port      = number
+      meta      = map(string)
+      tags      = list(string)
+      namespace = string
+      status    = string
+
+      node                  = string
+      node_id               = string
+      node_address          = string
+      node_datacenter       = string
+      node_tagged_addresses = map(string)
+      node_meta             = map(string)
+
+      cts_user_defined_meta = map(string)
+    })
+  )
+}
+`
+}
+
+func fileModuleContents() string {
```

Comment:
I'm not seeing that this method is used in this PR. Am I missing something or maybe there are future plans for it?

Reply:
No plans, just missed that when cleaning out leftovers. Didn't see it at the bottom. Thanks!